Structure_Engine/Query/OppositeNode.cs

```diff
@@ -30,12 +33,21 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("If the provided node belongs to the Bar, this method returns the Node on the end of the Bar. The node if checked if it belongs to the bar by comparing its Guid. If the provided node is neither start or end node, null is returned.")]
```

Comment:
If the provided node belongs to the Bar, this method returns the Node on the opposite end of the Bar. The Node is checked if it belongs to the Bar by comparing its Guid. If the provided Nodeis neither StartNode or EndNode, null is returned."